colossalai/fx/passes/algorithms/ckpt_solver_pofo.py

```diff
@@ -507,6 +508,7 @@ def solver_pofo(gm: ColoGraphModule,
     mem_limit -= parameter_size(gm)
 
     # prepare data
+    data = MetaTensor(data, fake_device=next(gm.parameters()).device)
```

Comment:
After last PR update, any data that is used in MetaInfoProp should first be wrapped in `MetaTensor`